pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -4415,7 +4415,6 @@ broken-packages:
   - dhall-check
   - dhall-docs
   - dhall-fly
-  - dhall-nix
```

Comment:
Since you've removed this line, the next time the automatic Hackage updater script runs, it will mark `dhall-nix` as unbroken.

You shouldn't have to do anything on your end, as long as you've checked it does actually compile correctly when enabling building packages marked broken.